aws/terminator/paas.py

```diff
@@ -20,3 +20,21 @@ def ignore(self) -> bool:
 
     def terminate(self):
         self.client.delete_event_source_mapping(UUID=self.id)
+
+
+class LambdaLayers(DbTerminator):
```

Comment:
Instead of making this a `DbTerminator` I would just make it a `Terminator` and use `self.instance['LatestMatchingVersion']['CreatedDate']` as the `created_time`.

Reply:
done